src/BBCodeParser.php

```diff
@@ -126,7 +126,7 @@ public function parse() {
 		$config = parse_ini_file('BBCodeParser.ini', true);
 		$options = $config['SSHTMLBBCodeParser'];
 
-		require_once(__DIR__ . '/../thirdparty/HTML_BBCodeParser2/BBCodeParser2.php');
```

Comment:
I'll PR it in separately if that's ok? for now I just needed it working (and it is) for forum upgrade and it's going to be replaced anyway